workLoggingSystem.html

```diff
@@ -295,7 +296,7 @@
                     <div class="row formComponent">
                         <div class="col">
                             <div class="row">
-                                <div class="col-md-4 menu">Estimated Time:</div>
+                                <div class="col-md-4 menu ch">Estimated Time:</div>
```

Comment:
giving class ch to all will turn all the options red in color, where as it will be difficult to identify which one is not passing validation. Please give all of them a unique id with reasonable name and focus to change the particular id only which has not passed validation.